README.md

```diff
@@ -17,12 +17,13 @@ Create a workflow `.yml` file in your `.github/workflows` directory. An [example
 - `ADMIN_PASSWORD` : Password of the admin
 - `CERTIFICATE_ENCODED` : Base64-encoded string with certificate private key
 - `CERTIFICATE_PASSWORD` : Password for the certificate
-- `APP_ID` : App ID of the Azure AD application to use for certificate authentication. If not specified, use the app specified in the 'CLIMICROSOFT365_AADAPPID' environment variable. If the environment variable is not defined, use the multitenant PnP Management Shell app
+- `APP_ID` : App ID of the Entra application to use for authentication. If not specified, use the app specified in the 'CLIMICROSOFT365_ENTRAAPPID' environment variable.
 - `TENANT` : ID of the tenant from which accounts should be able to authenticate. Use `common` or `organization` if the app is multitenant. If not specified, use the tenant specified in the `CLIMICROSOFT365_TENANT` environment variable. If the environment variable is not defined, it will use `common` as the tenant identifier
 - `CLI_VERSION` : Accepts `latest`, `next` or a specific version tag. Otherwise, installs the `latest` version when omitted
 
 All inputs are optional. But depending of the authentication type chosen, following pair of inputs will be necessary:
 
+- `authType` is `federatedIdentity`: only `APP_ID` and `TENANT` are necessary. (If not filled in, it will use the values from the environment variables `CLIMICROSOFT365_ENTRAAPPID` and `CLIMICROSOFT365_TENANT` respectively.)
```

Comment:
```suggestion
- `authType` is `federatedIdentity`: only `APP_ID` and `TENANT` are necessary. (If not filled in, it will use the values from the environment variables `CLIMICROSOFT365_ENTRAAPPID` and `CLIMICROSOFT365_TENANT` respectively).
```

Reply:
The new sentence starts within the brackets and the previous sentence already has a dot... So not implementing this suggestion